### ISSUE TITLE
Aquatic Sciences Open House - Activity ideas? 

### ISSUE THREAD

**Issue by grace-ac** [RobertsLab/resources#837]
SAFS and Oceanography are hosting the Aquatic Sciences Open House again this year!

Date: **May 2, 2020  1-4pm** 

Activity ideas?

**Comment by shellytrigg** [comment]
What if we did a lab demo talking about DNA methylation and sequencing to understand how shellfish are affected by their environment ? I know the concept is a little complicated but it would be cool to get people excited about this. For DNA part, we could do parts of this [strawberry DNA extraction demo](https://drive.google.com/file/d/0B48Fm4_B1Qqpdnd2QnFuR2hSSUxScGhORU11UDJvSWM5ZFVJ/view) that I've done in the past. We could have it be hands on with stuff ready made or we could have things pre-made like a cooking show. The coolest part of the demo is watching the DNA separate out when the alcohol is added. We could have some posters for talking about sequencing and about environmental influence on shellfish DNA methylation. We could show the nano pore and some data (would be cool to show live collection of data). 

Another idea I had (and done in the past) was making DNA double helix out of red vines/twizzlers, toothpicks and mini marshmallows as bases. And maybe use some other piece of candy to attach as methylation. And show how those marks can change under different conditions. This would be a booth in the lobby like we did last year.

I talked with Eileen and Arielle and they mentioned the labs and they mentioned the lab activities get pretty crowded with people constantly rotating in and out all day, and having about 5-7 minutes with different groups of people. If we were to do this, they recommended at least 3 people running the activity.
